test/pspm_split_sessions_test.m

```diff
@@ -168,8 +172,10 @@ function test_splitpoints(this, splitpoints)
       for i = 1:numel(newdatafile)
         if exist(newdatafile{i}, 'file')
           % test suffix and prefix
-          [~, info, ~] = pspm_load_data(newdatafile{i});
-          this.verifyEqual(info.duration, sess_dur(i), 'RelTol', 0.5);
+          if i > 1 && i < numel(newdatafile)
```

Comment:
I am not sure about why the situation `i==numel(newdatafile)` wants to be excluded?

Reply:
Both first and last session are special - no trimming at file start and end. However, I changed the code now to explicitly check session duration also for these sessions in latest commit.